<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>16</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>密码管理工具</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow
{
  
	background-color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QStackedWidget" name="wgtPages">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="wgtMain">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="wgtTop" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#wgtTop
{
  
	background-color: rgb(0, 0, 0);
    
	font: 14pt &quot;黑体&quot;;
}
#wgtTop QPushButton
{
  
	background-color: rgba(255, 255, 255, 0);
    
	color: rgb(255, 255, 255);

}
#wgtTop QPushButton:checked
{
   
	color: rgb(35, 164, 92);
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>16</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>241</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnAll">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>全部</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnRecent">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>最近</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnOften">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>常用</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>241</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="btnAdd">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>新宋体</family>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#btnAdd
{
	color: rgb(255, 255, 255);
    
	background-color: rgba(255, 255, 255, 0);
}</string>
             </property>
             <property name="text">
              <string>+</string>
             </property>
             <property name="icon">
              <iconset resource="../app.qrc">
               <normaloff>:/res/add.png</normaloff>:/res/add.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="wgtSearch" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QLineEdit" name="edtSearch">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#edtSearch
{
  
	background-color: rgb(20, 20, 20);
    border:none;
   
	color: rgb(255, 255, 255);
    border:2px groove rgba(255, 255, 255,0);border-radius:10px;padding:2px 4px;
}</string>
             </property>
             <property name="placeholderText">
              <string>输入查询内容</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lvData">
          <property name="styleSheet">
           <string notr="true">#lvData
{
  border:none;
  
	background-color: rgb(0, 0, 0);
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wgtEdit">
       <property name="styleSheet">
        <string notr="true">QLabel
{
  
	color: rgb(255, 255, 255);
}
QLineEdit
{
  border:none;
  
	color: rgb(255, 255, 255);
    
	background-color: rgb(25, 25, 25);
}
QPushButton
{
  
	background-color: rgb(25, 25, 25);
    
	font: 10pt &quot;黑体&quot;;
}
#btnDelete
{
  
	color: rgb(248, 84, 82);
}
#btnBack
{
   
	color: rgb(255, 255, 255);
}
#btnSave
{
   color: rgb(34, 164, 92);
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="wgtTop_2" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#wgtTop
{
  
	background-color: rgb(0, 0, 0);
    
	font: 14pt &quot;黑体&quot;;
}
#wgtTop QPushButton
{
  
	background-color: rgba(255, 255, 255, 0);
    
	color: rgb(255, 255, 255);

}
#wgtTop QPushButton:checked
{
   
	color: rgb(35, 164, 92);
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>16</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QToolButton" name="btnBack">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>新宋体</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#btnBack
{
	color: rgb(255, 255, 255);
    
	background-color: rgba(255, 255, 255, 0);
}</string>
             </property>
             <property name="text">
              <string>&lt;</string>
             </property>
             <property name="icon">
              <iconset resource="../app.qrc">
               <normaloff>:/res/add.png</normaloff>:/res/add.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>241</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>名    称：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edtName">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>21</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>用 户 名:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edtUserName">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>21</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>密    码：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edtPassword">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>21</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>备    注：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edtRemark">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>21</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSave">
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelete">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>259</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../app.qrc"/>
 </resources>
 <connections/>
</ui>
